src/ConnectedMode/Binding/BindingStrings.resx

```diff
@@ -117,31 +117,16 @@
   <resheader name="writer">
     <value>System.Resources.ResXResourceWriter, System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
   </resheader>
-  <data name="DownloadingQualityProfileProgressMessage" xml:space="preserve">
```

Comment:
Miscellaneous: all of the existing QP-related strings were in `Binding\BindingStrings.resx`.
I moved them to a new resource file `QualityProfiles\QualityProfileStrings.resx`.